examples/index.html

```diff
@@ -539,48 +538,26 @@ <h2 id="examples_by_props_label">Examples By Properties and States</h2>
-                <li><a href="combobox/grid-combo.html">Editable Combobox with Grid Popup</a></li>
-                <li><a href="dialog-modal/alertdialog.html">Alert Dialog</a></li>
                 <li><a href="dialog-modal/datepicker-dialog.html">Date Picker Dialog</a></li>
-                <li><a href="dialog-modal/dialog.html">Modal Dialog</a></li>
-                <li><a href="feed/feed.html">Feed</a></li>
-                <li><a href="grid/dataGrids.html">Data Grid</a></li>
-                <li><a href="grid/LayoutGrids.html">Layout Grid</a></li>
-                <li><a href="listbox/listbox-collapsible.html">Collapsible Dropdown Listbox</a></li>
-                <li><a href="listbox/listbox-grouped.html">Listbox  with Grouped Options</a></li>
-                <li><a href="listbox/listbox-rearrangeable.html">Listboxes with Rearrangeable Options</a></li>
-                <li><a href="listbox/listbox-scrollable.html">Scrollable Listbox</a></li>
-                <li><a href="menu-button/menu-button-actions-active-descendant.html">Actions Menu Button  Using aria-activedescendant</a></li>
-                <li><a href="menu-button/menu-button-actions.html">Actions Menu Button  Using element.focus()</a></li>
-                <li><a href="menu-button/menu-button-links.html">Navigation Menu Button</a></li>
                 <li><a href="menubar/menubar-editor.html">Editor Menubar</a></li>
                 <li><a href="menubar/menubar-navigation.html">Navigation Menubar</a></li>
-                <li><a href="radio/radio-activedescendant.html">Radio Group  Using aria-activedescendant</a></li>
-                <li><a href="radio/radio.html">Radio Group  Using Roving tabindex</a></li>
                 <li><a href="spinbutton/datepicker-spinbuttons.html">Date Picker Spin Button</a></li>
                 <li><a href="table/table.html">Table</a></li>
                 <li><a href="tabs/tabs-1/tabs.html">Tabs with Automatic Activation</a></li>
                 <li><a href="tabs/tabs-2/tabs.html">Tabs with Manual Activation</a></li>
                 <li><a href="toolbar/toolbar.html">Toolbar</a></li>
                 <li><a href="treegrid/treegrid-1.html">Treegrid Email Inbox</a></li>
-                <li><a href="treeview/treeview-1/treeview-1a.html">File Directory Treeview  Using Computed Properties</a></li>
-                <li><a href="treeview/treeview-1/treeview-1b.html">File Directory Treeview  Using Declared Properties</a></li>
-                <li><a href="treeview/treeview-2/treeview-2a.html">Navigation Treeview  Using Computed Properties</a></li>
-                <li><a href="treeview/treeview-2/treeview-2b.html">Navigation Treeview  Using Declared Properties</a></li>
               </ul>
             </td>
           </tr>
           <tr>
             <td><code>aria-labelledby</code></td>
             <td>
               <ul>
-                <li><a href="checkbox/checkbox-1/checkbox-1.html">Checkbox  (Two State)</a></li>
```

Comment:
This appears to be a bug; aria-labelledby is used and documented in the checkbox-1 example.